copyContentPlayer.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
```

Comment:
keep this file inside scripts folder. As we have all the scripts inside scripts folder.